packages/taquito-beacon-wallet/test/taquito-beacon-wallet.spec.ts

```diff
@@ -21,6 +21,12 @@ describe('Beacon Wallet tests', () => {
     expect(new MissingRequiredScopes([PermissionScope.OPERATION_REQUEST])).toBeInstanceOf(Error);
   });
 
+  it('Verify that requestPermissions is a function', async () => {
+    const wallet = new BeaconWallet({ name: 'testWallet' });
+    expect(typeof (await wallet.requestPermissions)).toEqual('function');
+    expect(await wallet.requestPermissions).toBeDefined;
```

Comment:
I'm not sure of the added value of validating that a function is a function. Can you explain?

Reply:
Yes, I will remove it; it was an assertion to get assertions working, but should have been cleaned up